pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -221,38 +220,7 @@ func (service *SecurityPolicyService) buildTargetTags(obj *v1alpha1.SecurityPoli
 
 // Todo, use the util basic func to generate basic tags
 func (service *SecurityPolicyService) buildBasicTags(obj *v1alpha1.SecurityPolicy, createdFor string) []model.Tag {
-	scopeOwnerName := common.TagValueScopeSecurityPolicyName
-	scopeOwnerUID := common.TagValueScopeSecurityPolicyUID
-	if createdFor == common.ResourceTypeNetworkPolicy {
-		scopeOwnerName = common.TagScopeNetworkPolicyName
-		scopeOwnerUID = common.TagScopeNetworkPolicyUID
-	}
-	tags := []model.Tag{
-		{
-			Scope: String(common.TagScopeVersion),
-			Tag:   String(strings.Join(common.TagValueVersion, ".")),
-		},
-		{
-			Scope: String(common.TagScopeCluster),
-			Tag:   String(getCluster(service)),
-		},
-		{
-			Scope: String(common.TagScopeNamespace),
-			Tag:   String(obj.ObjectMeta.Namespace),
-		},
-		{
-			Scope: String(common.TagScopeNamespaceUID),
-			Tag:   String(string(service.getNamespaceUID(obj.ObjectMeta.Namespace))),
-		},
-		{
-			Scope: String(scopeOwnerName),
-			Tag:   String(obj.ObjectMeta.Name),
-		},
-		{
-			Scope: String(scopeOwnerUID),
-			Tag:   String(string(obj.UID)),
-		},
-	}
+	tags := util.BuildBasicTags(getCluster(service), obj, service.getNamespaceUID(obj.ObjectMeta.Namespace))
```

Comment:
we can not remove scopeOwnerName, because inside BuildBasicTags, the parameter of obj is always the type of SecurityPolicy.So we have to use createdFor to decide we add networkpolicy owner tag or securitypolicy owner tag.

Reply:
I tested, it would duplicate TagValueScopeSecurityPolicyName  TagValueScopeSecurityPolicyUID, and test case would fail.

Reply:
For the duplicated case, you should fix it in BuildBasicTags function instead of removing it, otherwise, we can not support networkpolicy.

Reply:
I think it's better to separate  security policy and network policy, putting them together is not a good idea, since security policy itself is complicated enough.In the future it would grow more complex.
Currently and temporarily  I  remove the lines in BuildBasicTags though.

Reply:
Separation is a good idea. but our current design is fixed, we leverage SP logic to translate network policy to internal Security Policy, this is our design and save our time effort. we need to make trade off.